extensions/wikia/EditTagging/EditTaggingHooks.class.php

```diff
@@ -211,11 +211,11 @@ private static function tagRevisionIfGalleryEdit( $revisionId, $request ) {
 	 */
 	private static function getHandlers() {
 		$handlers = new ArrayObject();
-		$handlers->append( [ self, 'tagRevisionIfCategoryEdit' ] );
-		$handlers->append( [ self, 'tagRevisionIfGalleryEdit' ] );
-		$handlers->append( [ self, 'tagRevisionIfRTESourceEdit' ] );
-		$handlers->append( [ self, 'tagRevisionIfRTEWysiwygEdit' ] );
-		$handlers->append( [ self, 'tagRevisionIfSourceEdit' ] );
+		$handlers->append( [ self::class, 'tagRevisionIfCategoryEdit' ] );
+		$handlers->append( [ self::class, 'tagRevisionIfGalleryEdit' ] );
+		$handlers->append( [ self::class, 'tagRevisionIfRTESourceEdit' ] );
+		$handlers->append( [ self::class, 'tagRevisionIfRTEWysiwygEdit' ] );
+		$handlers->append( [ self::class, 'tagRevisionIfSourceEdit' ] );
```

Comment:
Seems like new PHP version doesn't like this.  Had to specify the class name to stop 500 errors during commenting